src/GitVersion.Core.Tests/Core/GitVersionExecutorTests.cs

```diff
@@ -398,16 +398,19 @@ public void WorkingDirectoryWithoutGit()
     [Test]
     public void WorkingDirectoryWithoutCommits()
```

Comment:
Why is the test `WorkingDirectoryWithoutCommits` suddenly successful?

Reply:
The idea was returning just the fallback version 0.0.0 to not break any build processes. But I have changed it back.